src/modules/cmdpal/Microsoft.CmdPal.UI.ViewModels/ListItemViewModel.cs

```diff
@@ -14,22 +14,70 @@ public partial class ListItemViewModel(IListItem model, TaskScheduler scheduler)
 
     // Remember - "observable" properties from the model (via PropChanged)
     // cannot be marked [ObservableProperty]
-    public ITag[] Tags { get; private set; } = [];
+    public List<TagViewModel> Tags { get; private set; } = [];
```

Comment:
We probably want this to be an `ObservableCollection` instead.

While it'd be great to use the tag model directly, that's probably not going to be practically feasible with the setup and thread issues. And if we even have them be more dynamic, we'd probably need to do some wrapping anyway.

But in the meantime, as an OC, if tags on items can change, it makes it easier for us to update the UI later in that sort of future scenario.

Reply:
Yea I think I instinctively avoided the `ObservableCollection` for the same reason as the other observable properties - I didn't want to necessarily deal with waiting to update them on the UI thread